### PR TITLE
milestone1: Survey-forn

### PR DESCRIPTION
**Pull request by GedeonTS**
# Survey-form

in this part we:
- created markup for the project
- added styling
- installed linters

### CODE REVIEW

**Review by GedeonTS — COMMENTED** [review]
# changes required :recycle: 

please check my comments below, make all required changes and make sure there are no linters errors.

submit for re-review when you are done.

Thanks :grin:

File: style.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    font-size: 16px;
+    background-color: blueviolet;
+    margin-left: 400px;
+    margin-top: 60px;
+    padding-left: 25px;
+
+
+;
+}
+#survey-form #input{
```
[the remark this answers is not in the document]
Comment (sja-thedude) [reply style.css:13]: Good job 👍 , You have reviewed it correctly.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    
+
+}
+#1{
+    padding-left:0;
+}
+
+html, body {min-height: 100%;}
+      
+body, div, form, input, select {
```
[the remark this answers is not in the document]
Comment (sja-thedude) [reply style.css:31]: Good job 👍 , You have reviewed it correctly.

**Review by sja-thedude — APPROVED** [review]
Hi @GedeonTS, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**